node/bindnode/api.go

```diff
@@ -0,0 +1,95 @@
+// Package bindnode provides an ipld.Node implementation via Go reflection.
```

Comment:
Maybe minor wishlist: add more of the pointer to the two entrypoints (`Prototype` and `Wrap`) to the package scope docs.  Some of the prose in the commit message is already quite good for this.

Reply:
... although on second thought, now that I look at how many exported symbols this package has (not many!), maybe this isn't very necessary.

Reply:
Good call, will do.

Reply:
Yeah, I was aiming at keeping the package godoc short, and hopefully make the (few) APIs pretty quick to navigate. I'll leave this out for now, and we can expand later if users find it confusing.